### PR TITLE
Post game clean up logic 

### PR DESCRIPTION
**Pull request by Luke-Bytes**

### CODE REVIEW

File: src/database/GameInstance.ts [hunk 1]
```diff
@@ -28,6 +28,22 @@ export class GameInstance {
...
+    this.settings = {
+      minerushing: undefined,
+      bannedClasses: undefined,
+      map: undefined,
+    };
+    this.teams = { RED: [], BLUE: [] };
+    this.mapVoteManager = undefined;
+    this.minerushVoteManager = undefined;
+  }
+
```
Comment (99-i) [inline src/database/GameInstance.ts:46]: how about when the game ends or whatever, `CurrentGameManager` calls `GameInstance.commitToDb()` or smth
and calls `this.currentGame = new GameInstance()`

File: src/database/PlayerInstance.ts [hunk 2]
```diff
@@ -5,6 +5,8 @@ import { Player } from "@prisma/client";
 // wrapper class for Player
 // todo bad naming
 export class PlayerInstance {
+  private static instances: PlayerInstance[] = [];
```
Comment (99-i) [inline src/database/PlayerInstance.ts:8]: this isn't needed, GameInstance already keeps track of all of the PlayerInstances.

File: src/database/PlayerInstance.ts [hunk 3]
```diff
@@ -25,6 +27,18 @@ export class PlayerInstance {
...
+  }
+
+  public static resetAll() {
+    PlayerInstance.instances = [];
+  }
+
+  public static removePlayerInstance(instance: PlayerInstance) {
+    PlayerInstance.instances = PlayerInstance.instances.filter(
+      (i) => i !== instance
+    );
```
Comment (99-i) [inline src/database/PlayerInstance.ts:41]: this isn't needed, GameInstance already keeps track of all of the PlayerInstances.

**Review by 99-i — COMMENTED** [review]
everything's good except adding that static array to PlayerInstance

File: src/database/PlayerInstance.ts (hunk 3 above)
Comment (Luke-Bytes) [reply src/database/PlayerInstance.ts:41]: Done

File: src/database/PlayerInstance.ts (hunk 2 above)
Comment (Luke-Bytes) [reply src/database/PlayerInstance.ts:8]: Done

File: src/database/GameInstance.ts (hunk 1 above)
Comment (Luke-Bytes) [reply src/database/GameInstance.ts]: add a note to do this in the new `resetGameInstance()` method

**Pull request merged**